builder/pipeline_test.cc

```diff
@@ -988,60 +1003,68 @@ TEST(pipeline, copied_result) {
 }
 
 TEST(pipeline, concatenated_result) {
-  // Make the pipeline
-  node_context ctx;
+  for (bool no_alias_buffers : {false, true}) {
```

Comment:
The drawback to this approach is that now we have two distinct tests that are being run as a single test, so if one fails you have to dive in to see which.  IMHO this would be better to move the guts into a helper function and then have two calls to it from separate tests. (It is unfortunate that GTest doesn't have a way to enable trivial variants like this without using Fixtures.)

Reply:
Agreed, this is #25. This pattern is already in use everywhere, I'm going to leave it to #25 to figure out.